app/models/snap_application.rb

```diff
@@ -54,6 +55,16 @@ class SnapApplication < ApplicationRecord
     where.not(id: Export.emailed_client.succeeded.application_ids)
   end)
 
+  def drive_status
+    if driver_applications.any? && driver_errors.empty?
+      :drive_in_flight
+    elsif driver_applications.any? && driver_errors.any?
```

Comment:
hat if an earlier drive had errors but a newer drive does not?

Reply:
What do you think about - upon re-drive we delete the existing driver errors?

Reply:
I like that, although it *could* prove challenging if we re-drive something and want to remember why it failed previously (eg: we manually update the data to make it pass but want to pass without manual intervention in the future). Perhaps we do not delete and look at the latest driver application instead?